ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -202,6 +202,13 @@ private void LoadSpreadsheetDocument(SpreadsheetDocument dSpreadsheet)
                 ColumnWidth = CalculateColumnWidth(8, Style.Font, this);
             }
 
+            // We don't want to trigger a NotifyWorksheetAdded on the workbook every time we add a new worksheet during
+            // the loading of the workbook, as that invalidates all the cached caculation values unnecessarily and can
+            // cause a large amount of time to be wasted during workbook loading. Large workbooks with many worksheets
+            // and many caculations were seeing 5-15% of load time being spent just being used marking calculations as
+            // dirty over and over again before this was disabled.
+            const bool notifyWorksheetsAdded = false;
```

Comment:
This could have just been an inline `false` passed into the 2 calls to `Add` below, but I wanted somewhere shared to add my explaining comment for future devs, so hoisted it into a variable